unittests/AST/ASTImporterGenericRedeclTest.cpp

```diff
@@ -666,11 +755,127 @@ ASTIMPORTER_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, FunctionTemplateSpec, ,
                                         ImportPrototypeThenProtoAndDefinition)
 
 ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Class, Conservative, ,
-                                            DoNotImportConflictingDefinition)
-ASTIMPORTER_ODR_INSTANTIATE_TYPED_TEST_CASE(RedeclChain, Class, Liberal, ,
-                                            ImportConflictingDefinition)
+                                            DontImportConflictingDefAfterDef)
```

Comment:
From this point the grouping of the test case instantiations has changed. Up to this point we grouped them by the test case, from this point we group by the Decl kind (e.g. Variable). Could you please indicate this in a comment? Also it would be useful to write in a comment that we use only the `.*conflicting.*` test cases.